### PR TITLE
Add % as single-line comment for Prolog lexer

### PR DESCRIPTION
**Pull request by jamesnvc**

### CODE REVIEW

File: lib/rouge/lexers/prolog.rb [hunk 1]
```diff
@@ -13,6 +13,7 @@ class Prolog < RegexLexer
       state :basic do
         rule /\s+/, Text
         rule /^#.*/, Comment::Single
+        rule /%.*/, Comment::Single
```
Comment (dblessing) [inline lib/rouge/lexers/prolog.rb]: Is a comment starting with `#` required to be at the beginning whereas `%` can be a trailing comment? I'm trying to understand if there should be a different between the two rules here as written.

Comment (dblessing) [reply lib/rouge/lexers/prolog.rb]: Will you also please add a comment example with `%` in `spec/visual/prolog`?

Comment (jamesnvc) [reply lib/rouge/lexers/prolog.rb]: As far as I know, `#` isn't actually a comment character at all in Prolog -- I suppose that's just there to make "shebangs" (`#!`) be highlighted properly?

`%` can indeed be either at the beginning of a line or trailing.

**Comment by dblessing** [comment]
Thanks alot @jamesnvc Sorry for the delay.

**Pull request merged**